src/GeositeFramework/sample_plugins/identify_point/main.js

```diff
@@ -31,6 +39,55 @@ define(["dojo/_base/declare", "framework/PluginBase"],
                     // Don't show this help on startup anymore, after the first time 
                     this.app.suppressHelpOnStartup(true);
                 }
+
+                // attach resize events to sample controls
+                var self = this;
+                _.each($(self.container).find("button.resize-btn"), function (el) {
+                    /* 'changeColor' and 'blink' are helper functions, to show how to chain
+                        other actions to the plugin resize event */
+                    var changeColor = function(color) {
+                        $(self.container).css({ background: color });
+                    };
+                    var blink = function (color, blinks) {
+                        var originalColor = $(el).css("background"),
+                            n = (blinks % 2 === 0) ? blinks : ++blinks;
+
+                        return _.compose(_.map(_.range(0, n), function (i) {
+                            return _.delay(changeColor, i * 100, (i % 2 === 0) ? color : originalColor);
+                        }));
+                    }
```

Comment:
I like this idea - I found the blink stuff a little complex.  We also don't need to show every option, but just the basic api usage - we'll have documentation in the wiki that will be more comprehensive.  I agree that it should be reduced and simplified.